pandas/core/arrays/integer.py

```diff
@@ -364,6 +364,9 @@ def __init__(self, values: np.ndarray, mask: np.ndarray, copy: bool = False):
             )
         super().__init__(values, mask, copy=copy)
 
+    def __neg__(self):
+        return type(self)(0 - self._data, self._mask)
```

Comment:
can you add a test for this directly in the IntegerArray tests

Reply:
and probably put this next to sum or _create_arithmetic_method?

could do `__pos__` and `__abs__` while at it?